docs/cli-reference/dfx-ledger.md

```diff
@@ -45,7 +45,7 @@ You can use the following optional flags with the `dfx ledger account-id` comman
 |------------------------------|--------------------------------------------------------|
 | `-h`, `--help`               | Displays usage information.                            |
 | `-V`, `--version`            | Displays version information.                          |
-| `--of-canister <ALIAS>`      | Alias of the canister controlling the account          |
+| `--of-canister <ALIAS>`      | Alias or name of the canister controlling the account  |
```

Comment:
Maybe I'm missing something, but doesn't `--of-principal` already provide this functionality? 

And if it didn't, wouldn't we be making this accept either the name(alias) or the principal?

```suggestion
| `--of-canister <ALIAS>`      | Alias or principal of the canister controlling the account  |
```

Reply:
You're completely right with the wording. No idea what I was thinking there...

Yes, `--of-principal` has this functionality already, but as a user when I want the account id of a _canister_ principal, I first think of using the `--of-canister` since I'm thinking first and foremost of the string as representing a _canister_.